src/app/components/MediaLoader/utils/buildSettings.test.ts

```diff
@@ -917,14 +921,14 @@ describe('buildSettings', () => {
       const hausaLiveRadioBlocks = [
         {
           type: 'liveRadio',
-          model: hausaLiveRadio?.content?.blocks,
+          model: hausaLiveRadio?.data?.mediaBlock?.[0]?.model,
         },
       ];
```

Comment:
We no longer need to do this, now that the data is returning the media block with type `liveRadio`. Can we remove this, and just pass in the mediaBlock from the data into buildSettings below?